lib/compare_task.rb

```diff
@@ -41,7 +41,7 @@ def render_html_comparison(description1, description2, scopes, options)
       end
     end
 
-    target = "/tmp/machinery-html-comparison"
+    target = File.join(Machinery::DEFAULT_CONFIG_DIR, "machinery-html-comparison")
```

Comment:
@cornelius got it. How about I put the full path, that should make it easier to spot what is meant to be done there. See next commit